packages/side-nav/src/SideNav.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { ulStyleOverrides } from './styles';
+
+const sideNavStyles = css`
```

Comment:
You might consider inlining this single-liner to improve readability (not that it's terrible now).